python/ct/crypto/verify.py

```diff
@@ -1,19 +1,23 @@
 """Verify CT log statements."""
 
-import hashlib
 import io
 import struct
 
+from ct.crypto import ecdsa_keys
 from ct.crypto import error
 from ct.crypto import merkle
-from ct.crypto import pem
+from ct.crypto import rsa_keys
```

Comment:
Any suggestions for better names? "{rsa,ecdsa}_keys" was the best I could come up with. Come to think of it, since the classes are now going to be called {Rsa,Ecdsa}Verifier, those module names are even less helpful. How about "verify_{rsa,ecdsa}"?